pandas/core/generic.py

```diff
@@ -5201,60 +5201,217 @@ def abs(self):
-              categorical objects use type object. See also the select_dtypes
-              documentation. eg. df.describe(include=['O'])
-            - If include is the string 'all', the output column-set will
-              match the input one.
+        percentiles : list of numbers, optional
+            The percentiles to include in the output. All should
+            fall between 0 and 1. The default is
+            ``[.25, .5, .75]``, which returns the 25th, 50th, and
+            75th percentiles.
+        include : None (default), 'all', or list of dtypes or strings, optional
+            A white list of data types to include in the result. Ignored
+            for `Series`. Here are the options:
+
+            - None (default). The result will include all numeric columns.
+            - 'all'. All columns on the input will be included in the output.
+            - A list of dtypes or strings. Limits the results to the
+              provided data types.
+              To limit the result to numeric types submit
+              ``numpy.number``. To limit it instead to categorical
+              objects submit the ``numpy.object`` data type. Strings
+              can also be used in the style of
+              ``select_dtypes`` (e.g. ``df.describe(include=['O'])``)
+        exclude : None (default) or a list of dtypes or strings, optional,
+            A black list of data types to omit from the result. Ignored
+            for Series. Here are the options:
+
+            - None (default). The result will exclude nothing.
+            - A list of dtypes or strings. Excludes the provided data types
+              from the result. To select numeric types submit
+              ``numpy.number``. To select categorical objects submit the data
+              type ``numpy.object``. Strings can also be used in the style of
+              ``select_dtypes`` (e.g. ``df.describe(include=['O'])``)
 
         Returns
         -------
-        summary: %(klass)s of summary statistics
+        summary: NDFrame of summary statistics
 
         Notes
         -----
-        The output DataFrame index depends on the requested dtypes:
 
-        For numeric dtypes, it will include: count, mean, std, min,
-        max, and lower, 50, and upper percentiles.
+        Analyzes both numeric and object series, as well
+        as DataFrame column sets of mixed data types.
 
-        For object dtypes (e.g. timestamps or strings), the index
-        will include the count, unique, most common, and frequency of the
-        most common. Timestamps also include the first and last items.
+        For numeric data, the result's index will include ``count``,
+        ``mean``, ``std``, ``min``, ``max`` as well as lower, ``50`` and
+        upper percentiles. By default the lower percentile is ``25`` and the
+        upper percentile is ``75``. The ``50`` percentile is typically the
+        same as the median.
```

Comment:
I was thinking of the alternative methods of returning medians when there are an even number of values that might result in differing expectations among users. But that's probably unnecessary. I will remove the qualification.